ui/helpers/utils/util.test.js

```diff
@@ -1017,4 +1017,17 @@ describe('util', () => {
       ).toBeFalsy();
     });
   });
+
+  describe('hexToText', () => {
```

Comment:
Can we add a test to cover the `hexToText` catch block?

Reply:
I just added this test case